### PR TITLE
remove frontend_package option

### PR DESCRIPTION
**Pull request by Lendemor**
### All Submissions:

- [x] Have you followed the guidelines stated in [CONTRIBUTING.md](https://github.com/pynecone-io/pynecone/blob/main/CONTRIBUTING.md) file?
- [x] Have you checked to ensure there aren't any other open [Pull Requests](https://github.com/pynecone-io/pynecone/pulls ) for the desired changed?

### Type of change
- [x] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] This change requires a documentation update

### Changes To Core Features:

- [x] Have you added an explanation of what your changes do and why you'd like us to include them?
- [x] Have you successfully ran tests with your changes locally?

### Description

Follow up to #1607

### CODE REVIEW

File: reflex/app.py [hunk 1]
```diff
@@ -493,16 +493,30 @@ def get_frontend_packages(self, imports: Dict[str, str]):
...
             for i in imports
             if i not in compiler.DEFAULT_IMPORTS.keys()
             and i != "focus-visible/dist/focus-visible"
             and "next" not in i
             and not i.startswith("/")
             and i != ""
-        ]
-        page_imports.extend(get_config().frontend_packages)
+        }
+        frontend_packages = get_config().frontend_packages
```
Comment (picklelo) [inline reflex/app.py:505]: Should we move these warnings to the `Config.__init__` method?

File: reflex/app.py [hunk 2]
```diff
@@ -493,16 +493,30 @@ def get_frontend_packages(self, imports: Dict[str, str]):
...
+        _frontend_packages = []
+        for package in frontend_packages:
+            if package in get_config().tailwind.get("plugins", []):  # type: ignore
+                console.warn(
+                    f"Tailwind packages are inferred from 'plugins', remove `{package}` from `frontend_packages`"
+                )
+                continue
```
Comment (picklelo) [inline reflex/app.py:512]: I don't think we should skip it for now - we should still install it if they specify it in the `frontend_packages`.

**Review by picklelo — COMMENTED** [review]
nice cleanup!

**Review by masenf — APPROVED** [review]

File: reflex/app.py (hunk 1 above)
Comment (Lendemor) [reply reflex/app.py:505]: We don't have access to the inferred packages in Config.__init__, so I'd say no.

File: reflex/app.py (hunk 2 above)
Comment (Lendemor) [reply reflex/app.py:512]: All it does is basically detect duplicates (packages declared in both `tailwing/plugins` and in `frontend_packages`)
so the package is installed at the end.

**Comment by nevdelap** [comment]
Hi there. When I've updated to this I get...

```
home 15:55:54 (0) ~/p/app> reflex init && reflex run
───────────────────────────────────────────────────────────────────────────────────────────────── Initializing app ─────────
[15:55:58] Initializing the web directory.
Success: Initialized app
──────────────────────────────────────────────────────────────────────────────────────────────── Starting Reflex App ───────
Compiling:  ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 100% 2/2 0:00:00
Traceback (most recent call last):
  File "/home/nev/miniconda3/envs/practicemylanguage/bin/reflex", line 8, in <module>
    sys.exit(cli())
             ^^^^^
  File "/home/nev/miniconda3/envs/practicemylanguage/lib/python3.11/site-packages/typer/main.py", line 214, in __call__
    return get_command(self)(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/nev/miniconda3/envs/practicemylanguage/lib/python3.11/site-packages/click/core.py", line 1157, in __call__
    return self.main(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/nev/miniconda3/envs/practicemylanguage/lib/python3.11/site-packages/click/core.py", line 1078, in main
    rv = self.invoke(ctx)
         ^^^^^^^^^^^^^^^^
  File "/home/nev/miniconda3/envs/practicemylanguage/lib/python3.11/site-packages/click/core.py", line 1688, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
                           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/nev/miniconda3/envs/practicemylanguage/lib/python3.11/site-packages/click/core.py", line 1434, in invoke
    return ctx.invoke(self.callback, **ctx.params)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/nev/miniconda3/envs/practicemylanguage/lib/python3.11/site-packages/click/core.py", line 783, in invoke
    return __callback(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/nev/miniconda3/envs/practicemylanguage/lib/python3.11/site-packages/typer/main.py", line 532, in wrapper
    return callback(**use_params)  # type: ignore
           ^^^^^^^^^^^^^^^^^^^^^^
  File "/home/nev/practicemylanguage/reflex/reflex/reflex.py", line 146, in run
    prerequisites.get_app()
  File "/home/nev/practicemylanguage/reflex/reflex/utils/prerequisites.py", line 113, in get_app
    app = __import__(module, fromlist=(constants.APP_VAR,))
          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/nev/practicemylanguage/app/practicemylanguage/practicemylanguage.py", line 17, in <module>
    app.compile()  # type: ignore
    ^^^^^^^^^^^^^
  File "/home/nev/practicemylanguage/reflex/reflex/app.py", line 634, in compile
    self.get_frontend_packages(all_imports)
  File "/home/nev/practicemylanguage/reflex/reflex/app.py", line 532, in get_frontend_packages
    if package in get_config().tailwind.get("plugins", []):  # type: ignore
                  ^^^^^^^^^^^^^^^^^^^^^^^^^
AttributeError: 'NoneType' object has no attribute 'get'
home 15:57:20 (1) ~/p/app>
```

Any ideas before I delve into it? Cheers.

**Comment by nevdelap** [comment]
I see it is a breaking change requiring a doco update. Do you have something to point me at regarding the changes I need to make?

**Comment by masenf** [comment]
@nevdelap we should fix this for the release! you shouldn't need to make code changes here. Thanks for helping us test.

**Comment by masenf** [comment]
Fix in #1761

**Pull request merged**